tests/accelerators/test_ddp.py

```diff
@@ -45,9 +44,18 @@ def test_multi_gpu_model_ddp_fit_only(tmpdir):
 
 
 @RunIf(min_gpus=2)
-def test_multi_gpu_model_ddp_test_only(tmpdir):
+def test_multi_gpu_model_ddp_fit_only_asfile(tmpdir):
+    _test_multi_gpu_model_ddp_fit_only(tmpdir, as_module=False)
+
+
+@RunIf(min_gpus=2)
+def test_multi_gpu_model_ddp_fit_only_asmodule(tmpdir):
+    _test_multi_gpu_model_ddp_fit_only(tmpdir, as_module=True)
```

Comment:
Can be parametrized. Same for the other tests added

```suggestion
@pytest.mark.parametrize('as_module', (False, True))
def test_multi_gpu_model_ddp_fit_only_asmodule(tmpdir, as_module):
    _test_multi_gpu_model_ddp_fit_only(tmpdir, as_module=as_module)
```